src/transition.js

```diff
@@ -133,7 +132,6 @@ export default class Transition {
     this.waitingForUpdate = false;
     this.tweets = null;
     this.currentTweetIndex = 0;
-    this.transitionIntervalId = null;
```

Comment:
Why was this removed? It's vital.

Reply:
@jungeunyoon because it is done already on the      this._stopTransitionTimer();. cheers